src/changelog.json

```diff
@@ -0,0 +1,30 @@
+[
```

Comment:
This duplicate file shouldn't be needed as it is already in src/assets/changelog.json. Or does this have a specific use here again? @aankitkumar31 ?